packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1559,7 +1568,10 @@ async function handler(
     organizer: {
       id: organizerUser.id,
       name: organizerUser.name || "Nameless",
-      email: organizerUser.email || "Email-less",
+      email:
+        eventType.useEventTypeDestinationCalendarEmail && destinationCalendar?.[0]?.primaryEmail
+          ? destinationCalendar[0].primaryEmail
+          : organizerUser.email || "Email-less",
```

Comment:
use the destination email as organizer email only when useEventTypeDestinationCalendarEmail is true